collections/_posts/2023-11-09-Elongated-structures.md

```diff
@@ -0,0 +1,355 @@
+- high curvatures can be effectively addressed by choosing a small-to-medium-sized heatmap width
+
+## Comparison to related curve representation
+
+Compare the method’s error to that of several related spatial curve representations
+
+### Mean squared error on curve points
+- A spatial representation of the curve is created by filtering the discrete curve points with a Gaussian low-pass filter
+- high approximation errors with increasing signal complexity (especially high curvature) → cause a shift in the distribution of the points, which affect the likelihood of points conforming to a Gaussian model
+
+### Gaussian-weighted Euclidean distance transform
+- represent a curve as a distance field by discretizing its points and using the Euclidean distance transform (EDT)
+- distances evaluated using a Gaussian probability density function
+- the observed approximation errors using this method are similar to those obtained with a proposed numerical approximation + smoothness increases as more sampling points are used
+- stair-step artifacts introduced due to the curve's discretization → can be problematic
+- for extremely large noise spread and very dense curve sampling, this representation might be preferred due to very efficient computation with linear time complexity
+
+### Gaussian-weighted BCE
+- BCE is typically used for optimizing binary segmentation problems
+- a discrete 2D convolution is performed between the curve points and the impulse response of the BCE term
+- each position is weighted using a Gaussian probability density function (pdf)
+- approximation error calculated by comparing resulting curve with corresponding 1D
+- similar to MSE variant, peak flattening and migration toward the inside of the curve (especially in high-curvature areas with substantial noise)
+- sparse sampling can also lead to larger errors in regions with little curvature
+- minor offsets between the original curve and the approximation result in significant errors (linked to the B-spline interpolation scheme)
+
+
+## CNN-based estimation error for different signal and heatmap configurations
+Signal simulation model and ASSD metric to assess the accuracy of these approximations (*Simulated signals of chest X-ray, 2D experiment)
+
+![](/collections/images/elongated_structures/fig_5.jpg)
+
+*The relation between a signal distribution and heatmap distance-dependent function*
+- weak correlation between the signal distribution and the distance-dependent function of the heatmap
+- Using a rectangular distribution result in worse reconstruction of the original curve (ASSD metric error up to 5px)
+- Distance-dependent functions like Gaussian and Raised Cosine work well for different signal configurations
+- the custom catheter signal is estimated with minimal error across heatmap configuration
+- strength of the signal has minimal impact
+
+*The influence of signal and heatmap width*
```

Comment:
I think you might be using bullet lists too much, not doing lists forces you to be more synthetic I think